src/ddo/Service.ts

```diff
@@ -8,11 +8,14 @@ export type ServiceType =
     | 'access'
     | 'compute'
     | 'workflow'
+    | 'nft-access'
+    | 'nft-sales'
```

Comment:
did-sales is pending here?

Reply:
True. I forgot about that. I will create a new issue to add proper `did-sales` support